crates/uv/tests/it/init.rs

```diff
@@ -929,6 +929,66 @@ fn init_script_file_conflicts() -> Result<()> {
     Ok(())
 }
 
+// Init script should not trash an existing shebang.
+#[test]
+fn init_script_shebang() -> Result<()> {
+    let context = TestContext::new("3.12");
+
+    let child = context.temp_dir.child("foo");
+    child.create_dir_all()?;
+
+    let contents = "#! /usr/bin/env python3\nprint(\"Hello, world!\")";
```

Comment:
Is this whitespace intentional? I hadn't seen that
```suggestion
    let contents = "#!/usr/bin/env python3\nprint(\"Hello, world!\")";
```

Reply:
I've always been in the habit, and apparently it's always been supported: https://unix.stackexchange.com/a/276845

Reply:
(I find it perturbing, but.. if that's what you like :D)